src/jit/lsra.cpp

```diff
@@ -3689,7 +3694,6 @@ void LinearScan::buildRefPositionsForNode(GenTree*                  tree,
     {
         lsraDispNode(tree, LSRA_DUMP_REFPOS, (produce != 0));
```

Comment:
Wow, somehow I did a really bad job with the merge. Those were gone, but came back. I've reworked it just a bit and will post before and after dump excerpts.

---
In reply to: [127828836](https://github.com/dotnet/coreclr/pull/12857#discussion_r127828836) [](ancestors = 127828836)